ToolBox/ZerodhaDownloader/ZerodhaDataDownloaderProgram.cs

```diff
@@ -117,6 +122,7 @@ public static void ZerodhaDataDownloader(IList<string> tickers, string market, s
                         foreach (var bar in history)
                         {
                             var linedata = new TradeBar(bar.TimeStamp, pairObject, bar.Open, bar.High, bar.Low, bar.Close, bar.Volume, timeSpan);
+                            linedata.Time = linedata.Time.ConvertTo(exchangeTimeZone, dataTimeZone);
```

Comment:
After the other changes believe `kite.GetHistoricalData` is returning in UTC? So maybe it should be something like `bar.TimeStamp.ConvertFromUtc(dataTimeZone)`